content/en/apps/guides/security/securely-onboarding-users-at-scale.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Securely onboarding users at scale"
+linkTitle: "Securely Onboarding Users"
+weight:
+description: >
+    How to securely create users, handle password changes and password breaches
+relatedContent: >
+    apps/guides/training/onboarding
+    apps/guides/database/couchdb-authentication
+    apps/tutorials/contact-and-users-1
+    apps/guides/data/users-bulk-load
+    apps/concepts/users
+    apps/concepts/access/#magic-links-for-logging-in-token-login
+
+---
+
+{{% pageinfo %}}
+The more secure a system is, the harder it is to use. Following this document will achieve the highest level of security. That said, implementers need to know when ease of use is more important than a more secure system. By reading this document you should be able to know when to make the "more secure" vs "easier to use" trade off. 
+
+No system is perfectly secure - be prepared to remediate a security breach!
+{{% /pageinfo %}}
+
+When a CHT deployment will support 100s or even 1000s of users, secure credential management becomes critical. Assuming there's a 10:1 ratio of 4-person families per CHW, this means there's potentially over 40,000 patients in a large CHT instance. Their data is sensitive and should never be viewed, or worse edited, by any unauthorized parties. By following best practices, being aware of and prepared for a worst case scenario of a password breach, patient data can be kept safe and CHWs can be kept online and able to deliver care.
+
+## Before starting
+
+### Secure devices
+
+It is assumed that the CHWs' devices are secure: they all employ disk encryption and require a password or PIN to unlock and use. Please see our [Securing Android Devices document]({{% ref "apps/guides/security/securing-android" %}}) for more information. As well, an [MDM]({{% ref "apps/guides/android/publishing#mobile-device-management" %}}) may be used to enforce disk encryption and device unlock protocols.
+
+### Secure administrative users
+
+CHT administrators have the ability to create and delete users and push new configurations to the CHT should take extra precautions in managing their password. They should use a [strong passphrase](https://en.wikipedia.org/wiki/Passphrase) (instead of a password) that is unique to their CHT login. They should use a [password manager](https://en.wikipedia.org/wiki/Password_manager) to store this password. 
+
+By not storing their password in [plaintext](https://en.wikipedia.org/wiki/Plaintext) and by using a strong passphrase, unauthorized users are more likely to be kept out of their CHT user. 
```

Comment:
This is hard to read - I think there's a double negative in there that's making it complex.

```suggestion
By following these steps, unauthorized people are less likely to be able to access administrator accounts. 
```